spotify_tensorflow/dataset.py

```diff
@@ -53,6 +53,21 @@ def parse_schema(cls, schema_path):
         schema = SchemaToFeatureSpec.parse_schema_file(schema_path)
         return SchemaToFeatureSpec.apply(schema), schema
 
+    @classmethod
+    def parse_schema_from_stats(cls, stats_path):
+        # type: (str) -> Tuple[Dict[str, Union[tf.FixedLenFeature, tf.VarLenFeature, tf.SparseFeature]], Schema]  # noqa: E501
+        """
+        Returns TensorFlow Feature Spec and parsed tf.metadata Schema for given tf.metadata
+        DatasetFeatureStatisticsList.
+
+        :param stats_path: tf.metadata DatasetFeatureStatisticsList path
+        """
+        import tensorflow_data_validation as tfdv
+        from .tf_schema_utils import SchemaToFeatureSpec
```

Comment:
Should we use an absolute import here?

Reply:
Yep, done. I was following the example from the `parse_schema` function.